scheduler/scheduler-api/src/main/resources/org/ow2/proactive/scheduler/common/xml/schemas/jobdescriptor/dev/schedulerjob.xsd

```diff
@@ -151,6 +159,16 @@
       </xs:simpleType>
     </xs:attribute>
   </xs:attributeGroup>
+  <xs:attributeGroup name="taskRetryDelay_j">
+    <xs:attribute name="taskRetryDelay" use="required">
+      <xs:annotation>
+        <xs:documentation>For each task, specifies how long to wait before restart the task if an error occurred.</xs:documentation>
```

Comment:
`For each task, it specifies how long to wait before restarting the task if an error occurred.` -> To be propagated to other occurrences.

Reply:
Done.